README.md

```diff
@@ -1,20 +1,17 @@
 # kbatch-papermill
 
-run notebooks with [papermill] on kubernetes via [kbatch], currently designed for [Destination-Earth GFTS](https://github.com/destination-earth/DestinE_ESA_GFTS).
+Currently designed for [Destination-Earth GFTS](https://github.com/destination-earth/DestinE_ESA_GFTS), it runs notebooks with [papermill] on Kubernetes via [kbatch].
```

Comment:
Actually, I thought that it was a good practice to display in the documentation the readme (since the latter usually aims to present the repository). During the build, I copy the current readme file in `docs/` to include in the documentation the last version (something which might be a bit _dirty_...) specifically to avoid any mismatch between what the README and the presentation section of the documentation describe.

I will remove that part and instead add a description of the package (very similar to the current readme) in `docs/`.